content/docs/ops/runbook/rotating-bosh.md

```diff
@@ -0,0 +1,194 @@
+
+```sh
+bosh-cli cloud-check -d ${deployment_name}
+```
+
+This command will check all the VMs found in the `${deployment_name}` and prompt
+you through the next steps. Select option `4` which will recreate and monitor
+the VMs as they each one spins up. Because these VMs are being recreated, **this
+process can take a while**. Monitor the recreation task and the platform for any
+issues.
+
+You can attach and detach from BOSH tasks at any time by referencing the task
+ID.
+
+```sh
+bosh-cli tasks -a -r | head
+```
+
+Once you have a list of task IDs, you can launch into it by attaching to the
+task.
+
+```sh
+bosh-cli task ${task_id}
+```
+
+You can detach from the task without canceling it with `Ctrl-C`.
+
+## Coordinate & monitor deployments
+
+There are two types of deployments that need to be coordinated here. All of the
+BOSH directors deploy using the previous BOSH director in the pipeline with the
+exception of the environment BOSH deployments (e.g. `development`, `staging`,
+`production`). The other type of dependency are deployments external to the
+`deploy-bosh` pipeline which use the BOSH credentials in either their secrets or
+credential files.
+
+### External deployments
+
+There are various pipelines that require BOSH credentials which will immediately
+be out of date once the BOSH deployment it targets and cause deployment failures
+and potentially cause downtime. All pipelines depending on BOSH must be paused
+and updated before unpausing them again.
+
+#### Finding deployments with BOSH dependencies
+
+Search GitHub for the `BOSH_TARGET` and `BOSH_ENVIRONMENT` with a `YAML`
+filetype to find all the different deployments that will need to be paused and
+updated after all the Bosh deployments are
+
+```bash
+open \
+  'https://github.com/search?l=YAML&q=user%3A18F+BOSH_TARGET&type=Code&utf8=%E2%9C%93' \
+  'https://github.com/search?l=YAML&q=user%3A18F+BOSH_ENVIRONMENT&type=Code&utf8=%E2%9C%93' \
+  'https://github.com/search?l=YAML&q=user%3A18F+upload-release&type=Code' \
+  'https://github.com/search?l=YAML&q=user%3A18F+bosh-errand&type=Code'
+```
+
+Once these pipelines are paused, update the values after all BOSH deployments
+have successfully deployed. Then unpause pipelines after you have successfully
+uploaded secrets or reflown pipelines.
+
+### BOSH deployments
+
+There are five BOSH deployments which are used to deploy cloud.gov. Master BOSH
+deploys Tooling BOSH which deploys Development, Staging, and Production BOSH.
+During a deployment of the BOSH director, BOSH will be unresponsive and will
+lead to errors in [any deployments that interact with the director](#external-deployments).
+Verify that these external deployments have been paused before proceeding with
+these steps.
+
+Besides certificates, all secrets in each BOSH deployment can be generated and
+replaced or can be updated from [rotated IAM users]({{< relref "docs/ops/runbook/rotating-iam-users.md" >}}).
+
+#### Validating BOSH deployments
```

Comment:
I was thinking this would at least be good to run just ensure that nothing is broken with the pipeline. Running the `common-releases-*` shouldn't be that big of a deal? wdyt?